src/fmu/dataio/scripts/create_case_metadata.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+
+"""
+
+Create FMU case metadata and register case on Sumo (optional).
+This script is intended to be run through an ERT HOOK PRESIM workflow.
+
+Script will parse global variables from the template location. If
+pointed towards the produced global_variables, fmu-config should run
+before this script to make sure global_variables is updated.
+
+"""
+import yaml
+import argparse
+import logging
+from pathlib import Path
+
+from ert_shared.plugins.plugin_manager import hook_implementation  # type: ignore
+from res.job_queue import ErtScript  # type: ignore
+
+from fmu.config._loader import FmuLoader
+from fmu.dataio import InitializeCase
+from fmu.sumo.uploader import SumoConnection, CaseOnDisk
```

Comment:
This seems to require that fmu.config and fmu.sumo is installed (i.e. add to requirements.txt)

Otherwise you need a lazy loading, i.e. a try: except on these packages

Reply:
Yes, will fix. Have done this in other versions of this script, should have been lazy loading here as well.

Reply:
Now lazy loaded